ApplicationCode/ProjectDataModel/RimWellPathTarget.cpp

```diff
@@ -61,6 +67,9 @@ RimWellPathTarget::RimWellPathTarget()
     m_hasTangentConstraintUiField.xmlCapability()->disableIO();
     CAF_PDM_InitField( &m_azimuth, "Azimuth", 0.0, "Azi(deg)", "", "", "" );
     CAF_PDM_InitField( &m_inclination, "Inclination", 0.0, "Inc(deg)", "", "", "" );
+
+    CAF_PDM_InitField( &m_lateralMDConnection, "LateralMD", 0.0, "Lateral Connection MD", "", "", "" );
```

Comment:
Consider rename to `Lateral Anchor Location MD`

Reply:
done